src/EnergyPlus/Construction.cc

```diff
@@ -712,7 +712,7 @@ void ConstructionProps::calculateTransferFunction(EnergyPlusData &state, bool &E
 
                     // As with the 1-D solution, we are accounting for the thermal mass
                     // of the half-node at the surface by adding it to the first row
-                    // of interior nodes at both sides of the this->  This is not
+                    // of interior nodes at both sides of the construction.  This is not
```

Comment:
OK this is kinda funny.  But yes, obvious /search/replace/g problem.  Thanks @shorowit